src/site/content/en/identity/passkey-form-autofill/index.md

```diff
@@ -18,168 +18,269 @@ tags:
 [Passkeys](https://developers.google.com/identity/passkeys) replace passwords
 and make user accounts on the web safer, simpler, easier to use. However, the
 transition from password-based to passkey-based authentication can complicate
-the user experience. Using form autofill to suggest passkeys can help
+the user experience. Leveraging form autofill to suggest passkeys can help
 create a unified experience.
 
 ## Why use form autofill to sign-in with a passkey?
 
-Using passkeys instead of passwords is a great way for websites to make their
-user accounts safer, simpler, easier to use. With a passkey, a
-user can sign in to a website or an app just by using fingerprint, face or
-device pin.
+With a passkey, a user can sign in to a website just by using fingerprint, face
```

Comment:
Just to check—I'm aware it was the same copy before as well—but is it meant to be a face pin or a device pin, because if not, an Oxford comma might come in handy here. I'm guessing it's implying face identification?

Reply:
It's about a fingerprint biometric sensor, face recognition or a device PIN. So PIN only applies to the device. I think Oxford comma sounds suitable in this case.

```suggestion
With a passkey, a user can sign in to a website just by using fingerprint, face,
```